hometask8/controllers/Auth.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var url = require('url');
+
+
+var Auth = require('./AuthService');
```

Comment:
Import name "Auth" is ambiguous whn importing from './AuthService' in the Auth.js file.
Consider renaming "Auth" import variable to "AuthService"